Makefile

```
@@ -295,6 +298,29 @@ endif
 
 pgadmin4-img-docker: pgadmin-img-build
 
+# Special case args: CCP_PGBOUNCER_VERSION
+pgbouncer-img-build: ccbase-image $(CCPROOT)/build/pgbouncer/Dockerfile
+	$(IMGCMDSTEM) \
+		-f $(CCPROOT)/build/pgbouncer/Dockerfile \
+		-t $(CCP_IMAGE_PREFIX)/crunchy-pgbouncer:$(CCP_IMAGE_TAG) \
+		--build-arg BASEOS=$(CCP_BASEOS) \
+		--build-arg BASEVER=$(CCP_VERSION) \
+		--build-arg PG_FULL=$(CCP_PG_FULLVERSION) \
+		--build-arg PG_MAJOR=$(CCP_PGVERSION) \
+		--build-arg PREFIX=$(CCP_IMAGE_PREFIX) \
+		--build-arg BOUNCER_VER=$(CCP_PGBOUNCER_VERSION) \
```

:pick: To stay consistent with pgAdmin, maybe?
```suggestion
		--build-arg PGBOUNCER_VER=$(CCP_PGBOUNCER_VERSION) \
```

---

Updated the pgbouncer docker file and the Makefile with above suggestion